src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationService.java

```diff
@@ -95,6 +96,7 @@ public void logNotificationFailure(
         parameters.put(DEFENDANT_NAME, claim.getClaimData().getDefendant().getName());
         parameters.put(FRONTEND_BASE_URL, notificationsProperties.getFrontendBaseUrl());
         parameters.put(CLAIM_REFERENCE_NUMBER, claim.getReferenceNumber());
+        parameters.put("ccjRequestedDate", Formatting.formatDate(claim.getCountyCourtJudgmentRequestedAt()));
```

Comment:
1. why not const?
2. I'm not sure if the string value is consistent. On the other hand all dates have weird, not-so-consistent names

Reply:
1. this is the only place this is being used.